### PR TITLE
Add V5+ and V5 nano documents 

### PR DESCRIPTION
**Pull request by hamishwillee**
Replaces #https://github.com/PX4/px4_user_guide/pull/520

### CODE REVIEW

**Comment by hamishwillee** [comment]
@cuhome Can you please sanity check my changes. In particular the introduction to the two autopilot pages where I have removed the images and replaced them with "feature lists".
There is a rendered version of the docs here: https://hamishwillee.gitbooks.io/ham_px4userguide/content/v/cuav_5plus_hw/en/flight_controller/cuav_v5_plus.html

Questions:
1. Do the boards come with the Neo GPS? Or is this separate purchase (is not completely clear from docs). If it is purchased, where from?
2. The branding appears to be "v5 nano", not "v5 Nano" - that is correct right?

When you've reviewed, let me know and I can merge. I may have minor further changes but this is good enough to go out.

**Comment by cuhome** [comment]
图片

File: en/assembly/quick_start_cuav_v5_nano.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+## SD CarD (Optional)
+
+The factory has completed inserting the SD card, you do not need to operate.
+
+
+## Motors
+
+Motors/servos are connected to the MAIN ports in the order specified for your vehicle in the [Airframes Reference](../airframes/airframe_reference.md).
```
Comment (hamishwillee) [inline en/assembly/quick_start_cuav_v5_nano.md]: Note, for nano this is only the MAIN ports. So it won't work for something that uses an AUX port. We should perhaps say that in the introduction.

**Comment by cuhome** [comment]
CUAV V5+ Manual：http://manual.cuav.net/V5-Plus.pdf
CUAV V5+ docs：http://doc.cuav.net/flight-controller/v5-autopilot/en/v5+.html
FMUv5 reference design pinout：https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165)
CUAV Github：https://github.com/cuav
Base board design reference ：https://github.com/cuav/hardware/tree/master/V5_Autopilot/V5%2B/V5%2B_BASE

File: en/assembly/quick_start_cuav_v5_nano.md (hunk 1 above)
Comment (cuhome) [reply en/assembly/quick_start_cuav_v5_nano.md]: Please help me modify it.

**Comment by cuhome** [comment]
The motherboard is not equipped with Neo GPS and needs to be purchased separately, I will introduce it separately.
It is V5 nano is not V5 Nano

**Comment by hamishwillee** [comment]
cuhome Cheers. The link to base board above gives me a 404 error. Can this be opened up?

I think this is ready to merge. OK?

**Comment by cuhome** [comment]
图片

**Comment by cuhome** [comment]
图片

**Comment by cuhome** [comment]
The above two images have errors, no other problems

**Comment by hamishwillee** [comment]
@cuhome I don't see the problems with the two images above.

**Comment by hamishwillee** [comment]
@cuhome Note again, I do not understand what problems you see in the diagrams above. 
https://github.com/PX4/px4_user_guide/pull/521#issuecomment-511716968
https://github.com/PX4/px4_user_guide/pull/521#issuecomment-511717313

**Pull request merged**